public/index.html

```diff
@@ -6,11 +6,11 @@
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="theme-color" content="#000000" />
         <meta
-            name="description"
-            content="Web site created using create-react-app"
+            name="Pokemon Card Collector"
```

Comment:
this needs to still be description. it's like a key value pair
the meta name is description and the meta content is the description